code/datums/craft/recipes/tools.dm

```diff
@@ -77,6 +77,26 @@
 		list(QUALITY_ADHESIVE, 15, 20)
 	)
 
+// For when you want everything in a single arm.
+/datum/craft_recipe/tool/toolimplant
+	name = "Multitool implant"
+	result = /obj/item/organ_module/active/multitool/improvised
+	steps = list(
+		list(/obj/item/storage/toolbox, 1),
+		list(/obj/item/circuitboard, 1),
+		list(/obj/item/organ/external/robotic, 1),
```

Comment:
```suggestion
		list(/obj/item/organ/external/robotic/junktech/, 1),
```
IMO it makes more sense for a makeshift tool implant with improvised tools to take an improvised junk organ

Reply:
The way it is currently set up, _**any**_ robotic limb, not only junktech, but any, can be used.
Restricting it to only junktech would just add even more step, not to mention that people would probably just make junktech limbs anyway.

Reply:
Fair, I just wish it was the same for the junktech robot to make since it requires a **borg** arm and thus uncraftable through scavenging as most machines drop implants and not borg parts...